React_Notes_Block/src/components/notes/InputChange.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import PropTypes from 'prop-types';
+import { Slide } from '@material-ui/core';
+import useStyles from '../../pages/mainNotes/listNotes/styled';
+import { ButtonSaveChanges, ButtonDelete } from './ActionButtons';
+
+const InputChange = ({
+  isChange,
+  description,
+  saveEditedNote,
+  id,
+  deleteNotes,
+  index,
+  currentState,
+}) => {
+  const [newInputState, setNewInputState] = useState(description);
```

Comment:
Is there any cases for setting old state?
No need to write new, and also inout, here we can set 100000 inputs and what will be the differences?
```suggestion
  const [updatedDescription, setDescription] = useState(description);
```

Reply:
Renamed its